quickstarts/k6-prometheus/config.yml

```diff
@@ -0,0 +1,49 @@
+slug: k6-prometheus
+
+title: k6 Prometheus (remote write)
+
+description: |
+  ## Quickstart for k6 Load Test Monitoring
+
+  k6 can send performance metrics to New Relic through the New Relic StatsD integration. By integrating New Relic and k6, you can visualize and correlate test result data, alert on performance data, and share advanced load testing results and their KPI's with your teams. View alongside all of your other telemetry data in New Relic for added insights.
+  Installation is as simple as running the NR StatsD integration and including NR StatsD as an output in your k6 load testing script.
+
+  This quickstart includes dashboards for the following k6 charts:
+  - Number of virtual users
+  - Max, median, and average Request Durations
+  - Rate of requests (per second)
+  - Histogram bucketing requests
+  - Data sent and data received
+  - Rate of change in the number of requests
+  - Blocked requests
+  - Response timings (90th percentile)
+  - List of all available k6 metrics
+
+summary: |
+  Send k6 performance metrics to New Relic through the New Relic StatsD integration.
+
+level: Community
+icon: logo.png
+
+authors:
+  - Kinsey
+
+keywords:
+  - k6
+  - grafana
+  - load testing
+  - statsD
+
+documentation:
+  - name: Set up k6 to send data to New Relic
+    url: https://k6.io/docs/results-output/real-time/prometheus-remote-write/
+    description: Information on how to set up k6 to send data to New Relic, including all configuration options from k6.
+  - name: Set up your Prometheus remote write integration
+    url: https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/
+    description: Information on how to set up your Prometheus remote write integration with New Relic. This will require you to set up these two environment variables K6_PROMETHEUS_RW_SERVER_URL=https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=k6 and K6_PROMETHEUS_RW_BEARER_TOKEN=NEW_RELIC_LICENSE_KEY
```

Comment:
```suggestion
    description: Information on how to set up your Prometheus remote write integration with New Relic. This will require you to set up these two environment variables: `K6_PROMETHEUS_RW_SERVER_URL=https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=k6` and `K6_PROMETHEUS_RW_BEARER_TOKEN=NEW_RELIC_LICENSE_KEY`
```

Reply:
updated description (colon within description is however not possible)